code/game/gamemodes/miniantags/abduction/machinery/experiment.dm

```diff
@@ -181,8 +183,8 @@
 				to_chat(H, "<span class='warning'>You feel intensely watched.</span>")
 		sleep(5)
 		to_chat(H, "<span class='warning'><b>Your mind snaps!</b></span>")
-		var/objtype = pick(subtypesof(/datum/objective/abductee/))
-		var/datum/objective/abductee/O = new objtype()
+		var/objtype = (prob(72) ? /datum/objective/abductee/random : pick(subtypesof(/datum/objective/abductee/)))
+		var/datum/objective/abductee/O = new objtype() //45 possible objectives as of the time of this. 40 (88%) are under random. 90% - [1/6 * 100]% ~= 73%
```

Comment:
Not really, I was just blindly putting the information out there. Your math seems right.

Another way to do this would be pick(subtypesof(/datum/objective/abductee) - /datum/objective/abductee/random). I'm not saying you have to do that way though, doing it the way you did is fine. Just leaving out more options for future considerations.

Also upon consideration, I would suggest removing the comment. People reading this a year from now won't know what it's referring to. Plus keeping the current odds isn't really a goal in and of itself.